app/views/admin/gias/home/index.html.erb

```diff
@@ -0,0 +1,30 @@
+<% content_for :title, "GIAS changes to schools" %>
+
+<h1 class="govuk-heading-xl">GIAS changes to schools</h1>
+
+<h2 class="govuk-heading-l govuk-!-font-size-27">
```

Comment:
I don't think we want to set the font size explicitly :)

Reply:
The views are only placeholders, this was lifted from GHwT where the designer wanted it like that!

Reply:
Removed.